UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -828,8 +846,333 @@ public IEnumerator ProcessDNAPayload(DNAMutationData InDNAMutationData)
+					 || bestSlot.NamedSlot == NamedSlot.storage04 || bestSlot.NamedSlot == NamedSlot.storage05 || bestSlot.NamedSlot == NamedSlot.storage06
+					  || bestSlot.NamedSlot == NamedSlot.storage07 || bestSlot.NamedSlot == NamedSlot.storage08 || bestSlot.NamedSlot == NamedSlot.storage09
+					   || bestSlot.NamedSlot == NamedSlot.storage10 || bestSlot.NamedSlot == NamedSlot.storage11 || bestSlot.NamedSlot == NamedSlot.storage12
+						|| bestSlot.NamedSlot == NamedSlot.storage13 || bestSlot.NamedSlot == NamedSlot.storage14)) // better don`t put fake clothes into storages
+					{
+						placed = false;
+					}
+
+
+
+					var itemName = fakeClothes.GetComponent<ItemAttributesV2>().InitialName;
+
+					if (!placed)
+					{
+						_ = Despawn.ServerSingle(fakeClothes);
+						continue;
+					}
+					//making items absolutely useless
+					fakeClothes.GetComponent<ItemAttributesV2>().IsFakeItem = true;
+
+					if (fakeClothes.TryGetComponent<WearableArmor>(out var armor))
+					{
+						foreach (var bodyArmr in armor.ArmoredBodyParts)
+						{
+							bodyArmr.Armor.Melee = 0;
+							bodyArmr.Armor.Bullet = 0;
+							bodyArmr.Armor.Laser = 0;
+							bodyArmr.Armor.Energy = 0;
+							bodyArmr.Armor.Bomb = 0;
+							bodyArmr.Armor.Rad = 0;
+							bodyArmr.Armor.Fire = 0;
+							bodyArmr.Armor.Acid = 0;
+							bodyArmr.Armor.Magic = 0;
+							bodyArmr.Armor.Bio = 0;
+							bodyArmr.Armor.Anomaly = 0;
+							bodyArmr.Armor.DismembermentProtectionChance = 0;
+							bodyArmr.Armor.StunImmunity = false;
+							bodyArmr.Armor.TemperatureProtectionInK = new Vector2(283.15f, 283.15f + 20);
+							bodyArmr.Armor.PressureProtectionInKpa = new Vector2(30f, 300f);
+
+						}
+					}
+					if (fakeClothes.TryGetComponent<IDCard>(out var card))
+					{
+						var cardJobName = card.GetJobTitle();
+						var cardRegName = card.RegisteredName;
+						card.ServerChangeOccupation(OccupationList.Instance.Get(JobType.ASSISTANT), true, true);
+
+						//card.SyncJobTitle("", cardName);
+						//card.SyncName("", cardRagName);
+						card.ServerSetRegisteredName(cardRegName);
+						card.ServerSetJobTitle(cardJobName);
+						for (int i = 0; i < card.currencies.Length; i++)
+						{
+							card.currencies[i] = 0; // removing all currencies on card to be sure
+						}
+					}
+					else if (fakeClothes.TryGetComponent<Headset>(out var headset))
+					{
+						Destroy(headset);
+						//Destroy(headset);
+						//headset.IsPowered = false;
+						//headset.EncryptionKey = EncryptionKeyType.None;
+						//headset.EmmitableSignalData.Clear();
+					}
+					else if (fakeClothes.TryGetComponent<NightVisionGoggles>(out var nightVision))
+					{
+						Destroy(nightVision);
+					}
+					else if (fakeClothes.TryGetComponent<PrescriptionGlasses>(out var glasses))
+					{
+						Destroy(glasses);
+					}
+					if (fakeClothes.TryGetComponent<ReagentContainer>(out var container))
+					{
+						Destroy(container);
```

Comment:
We don't use `Destroy()`
We use `Despawn` and `Spawn` classes to network objects.